docs/src/examples.rst

```diff
@@ -14,205 +14,216 @@
 
-   class MyStack extends cdk.Stack {
-     constructor(parent: cdk.App, name: string, props?: cdk.StackProps) {
-       super(parent, name, props);
+Step 2: Add the |EC2| and |ECS| Packages
+----------------------------------------
 
-       const vpc = new ec2.VpcNetwork(this, 'VPC');
+Install support for |EC2| and |ECS|.
 
-       new rds.DatabaseCluster(this, 'MyRdsDb', {
-         defaultDatabaseName: 'MyAuroraDatabase',
-         masterUser: {
-           username: 'admin',
-           password: '123456'
-         },
-         engine: rds.DatabaseClusterEngine.Aurora,
-         instanceProps: {
-           instanceType: new ec2.InstanceTypePair(ec2.InstanceClass.Burstable2, ec2.InstanceSize.Small),
-           vpc: vpc,
-           vpcPlacement: {
-             subnetsToUse: ec2.SubnetType.Public
-           }
-         }
-       });
-     }
-   }
+.. tabs::
 
-   const app = new cdk.App();
+    .. group-tab:: TypeScript
 
-   new MyStack(app, 'MyStack');
+        .. code-block:: sh
 
-   app.run();
+            npm install @aws-cdk/aws-ec2 @aws-cdk/aws-ecs
 
-.. _creating_s3_example:
+.. _creating_ecs_l2_example_3:
 
-Creating an |S3| Bucket
-=======================
+Step 3: Create a Fargate Service
+--------------------------------
 
-The following example creates the |S3| bucket **MyS3Bucket** with server-side KMS
-encryption provided by |S3|.
+There are two different ways of running your container tasks with |ECS|:
 
-.. code-block:: js
+- Using the **Fargate** launch type,
+  where |ECS| manages the physical machines that your containers are running on for you
+- Using the **EC2** launch type, where you do the managing, such as specifying autoscaling
 
-   import s3 = require('@aws-cdk/aws-s3');
-   import cdk = require('@aws-cdk/cdk');
+The following example creates a Fargate service running on an ECS cluster fronted by an internet-facing
+application load balancer.
 
-   class MyStack extends cdk.Stack {
-     constructor(parent: cdk.App, name: string, props?: cdk.StackProps) {
-       super(parent, name, props);
+.. tabs::
 
-       new s3.Bucket(this, 'MyBucket', {
-         bucketName: 'MyS3Bucket',
-         encryption: s3.BucketEncryption.KmsManaged
-       });
-     }
-   }
+    .. group-tab:: TypeScript
 
-   const app = new cdk.App();
+        Add the following import statements:
 
-   new MyStack(app, 'MyStack');
+        .. code-block:: typescript
 
-   app.run()
+            import ec2 = require('@aws-cdk/aws-ec2');
+            import ecs = require('@aws-cdk/aws-ecs');
+
+        Add the following code to the end of the constructor:
+
+        .. code-block:: typescript
+
+            const vpc = new ec2.VpcNetwork(this, 'MyVpc', {
+              maxAZs: 3 // Default is all AZs in region
+            });
+
+            const cluster = new ecs.Cluster(this, 'MyCluster', {
+              vpc: vpc
+            });
+
+            // Add capacity to it
+            cluster.addDefaultAutoScalingGroupCapacity({
```

Comment:
Don't need capacity if you're starting a Fargate task

Reply:
Right. I saw your comment to Paul in issue #1279.